runtime/onert/core/include/exec/Execution.h

```diff
@@ -24,8 +24,11 @@
 #include "ir/Layout.h"
 #include "exec/IExecutor.h"
 #include "IODescription.h"
+#include "../../api/src/nnfw_api_internal.h"
```

Comment:
Could you modify the relative path?

Reply:
I think you can remove this header if you use ir::DataType.